### ISSUE TITLE
ResizableBox can be resized only once with touch screen

### ISSUE THREAD

**Issue by toddlxt** [react-grid-layout/react-resizable#32]
Hi STRML,
    First I want to thank you for your great work.
    I'm using react-resizable. It works fine with mouse events but ResizableBox can be resized only once with touch screen(tested Chrome49+Windows10, and safari+iOS 9.3.1). I traced the error and it happens in Resizable.js line 125 and 126. When a touch event is ended, deltaX and deltaY are NaN. I simply added some code to convert NaN to 0, then it works fine.
    Another issue is that the resize event doesn't prevent default touch event. When I use a touch screen to resize ResizableBox, the page scrolls when the box is been resizing. I used 'react-tappable'(<Tappable preventDefault></Tappable>) to wrap the ResizableBox, then it works very well with touch screen. I think it's better if you can add a property to ResizableBox which can prevent default touch event.

**Comment by STRML** [comment]
Closing - this is related to #209 (touch resize issues). See that issue for status.
